flow/flow.yaml

```diff
@@ -29,7 +29,27 @@ components:
           minItems: 1
           items:
             $ref: './packet-headers/header.yaml#/components/schemas/Flow.Header'
+          x-status:
+            status: deprecated
+            information: |- 
+              Use flow.ingress_packet instead.
           x-field-uid: 2
+        ingress_packet:
+          description: |-
+            Ingress packet
+          type: array
+          minItems: 1
+          items:
+            $ref: './packet-headers/header.yaml#/components/schemas/Flow.Header'
+          x-field-uid: 8
+        egress_packet:
+          description: |-
+            Egress packets
```

Comment:
At least, we need to clarify that in the Egress packet structure, header field value choices are not to be configured. Only attribute to be configured is "metric_tags"